imgprune/entrypoint.sh

```diff
@@ -9,85 +15,52 @@ req_env_var GCPJSON GCPNAME GCPPROJECT
 
 gcloud_init
 
-# For safety's sake + limit nr background delete processes
-PRUNE_LIMIT=20
-THEFUTURE=$(date --date='+1 hour' +%s)
-TOO_OLD='30 days ago'
-THRESHOLD=$(date --date="$TOO_OLD" +%s)
+# Set this to 1 for testing
+DRY_RUN="${DRY_RUN:-0}"
+# For safety's sake limit nr deletions
+DELETE_LIMIT=10
+ABOUTNOW=$(date --iso-8601=date)  # precision is not needed for this use
 # Format Ref: https://cloud.google.com/sdk/gcloud/reference/topic/formats
-FORMAT='value[quote](name,selfLink,creationTimestamp,labels)'
+# Field list from `gcloud compute images list --limit=1 --format=text`
+FORMAT='value[quote](name,selfLink,deprecated.state,deprecated.deleted,labels)'
 # Required variable set by caller
 # shellcheck disable=SC2154
 PROJRE="/v1/projects/$GCPPROJECT/global/"
-RECENTLY=$(date --date='3 days ago' --iso-8601=date)
 # Filter Ref: https://cloud.google.com/sdk/gcloud/reference/topic/filters
-# shellcheck disable=SC2154
-FILTER="selfLink~$PROJRE AND creationTimestamp<$RECENTLY"
+FILTER="selfLink~$PROJRE AND deprecated.state=OBSOLETE AND deprecated.deleted>$ABOUTNOW"
```

Comment:
I'm finding it impossible to find examples of `deprecated.deleted` (or `deprecated.anything`), and I've spent far too long on this small piece already, but should that `>` (greater-than) be `<` (less-than)? If it's correct as it is, could you maybe add a comment explaining what `deprecated.deleted` means? And maybe add a corresponding comment in `imgobsolete/entrypoint.sh#L101` if it somehow corresponds to the `--delete-in=30d` therein?